examples/auth-generate-jwt/react/src/App.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
I wouldn't look too closely at the `auth-*` projects. They're not in scope to support just yet.